the-experts-fe/src/components/contractor/contractor.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./contractor.css";
+export default function Contractor({ contractor }) {
+  return (
+    <div className="card-transition">
+      <div className="contractor-card">
+        <div className="contractor-img">
+          <img
+            src="https://www.w3schools.com/bootstrap4/img_avatar3.png"
+            alt=""
```

Comment:
Why not add an `alt` value using the contractors name/job for better accessibility? eg:

```
alt={`Profile image of ${contractor.cont_name}, the ${contractor.job}`}
```